docs/modules/ROOT/pages/running-commodore.adoc

```diff
@@ -45,7 +45,47 @@ commodore() {
 == macOS X
 
 On macOS with Docker for Mac mounting the SSH agents socket into a container doesn't work.
-Instead your SSH key needs to be mounted into the container.
+Instead you need to mount as magic path that's provided by Docker for Mac.
+
+[NOTE]
+====
+This only works for the stock ssh-agent coming along with macOS.
+If you use any other agent, you are out of luck.
+Docker for mac doesn't support mounting sockets.
+====
+
+[CAUTION]
+====
+That magic socket path belongs to root.
+One must a container with `--user=0` in order to access it.
+In consequence, the files created by Commodore will also belong to root.
+If you do want to edit them on the host, you need to run `chown -R $UID .` first.
+====
+
+[source,bash]
+----
+commodore() {
+  docker run \
+    --interactive=true \
+    --tty \
+    --rm \
+    --user="$(id -u)" \
+    --env COMMODORE_API_URL=$COMMODORE_API_URL \
+    --env COMMODORE_GLOBAL_GIT_BASE=$COMMODORE_GLOBAL_GIT_BASE \
+    --env COMMODORE_API_TOKEN=$COMMODORE_API_TOKEN \
+    --env SSH_AUTH_SOCK=/tmp/ssh_agent.sock \
+    --volume "/run/host-services/ssh-auth.sock:/tmp/ssh_agent.sock" \
+    --volume "${HOME}/.ssh/config:/app/.ssh/config:ro" \
+    --volume "${HOME}/.gitconfig:/app/.gitconfig:ro" \
+    --volume "${PWD}:/app/data" \
+    --workdir /app/data \
+    --user=0 \
```

Comment:
This is probably conflicting/overriding with line 72 (`--user="$(id -u)"`), probably remove line 72?

Also, it's possible to set the group as well with `--user=0:<your-user>`, would something with group membership also solve file permissions?

Reply:
I tested that one again. I forgot another Docker for Mac magic: your file permissions are always correct :tm:. That is, their owner will always be your user.

I fixed the duplicate `--user` option.